unbalanced_dataset/base.py

```diff
@@ -181,3 +182,25 @@ def fit_sample(self, X, y):
         """
 
         return self.fit(X, y).sample(X, y)
+
+
+    def transform(self, X):
```

Comment:
@chkoar I added a `transform` function since that when using `transform` it seems required to have even if the function does not do anything. 

Furthermore, in `transform` in the pipeline module, you defined the behaviour such that the sampling will be discarded. Is that the behaviour that you wish?